### PR TITLE
support for clang 7

### PR DESCRIPTION
**Pull request by cosunae**
fix for supporting clang 7.0 from Ralf

### CODE REVIEW

**Comment by twicki** [comment]
launch jenkins

File: src/gtclang/Frontend/GTClangASTConsumer.cpp [hunk 1]
```diff
@@ -276,7 +276,14 @@ void GTClangASTConsumer::HandleTranslationUnit(clang::ASTContext& ASTContext) {
   // Write file to disk
   DAWN_LOG(INFO) << "Writing file to disk... ";
   std::error_code ec;
+#if  __clang_major__ == 6
```
Comment (cosunae) [inline src/gtclang/Frontend/GTClangASTConsumer.cpp]: this is not working since we are compiling with gcc, we have to do this via cmake

**Comment by twicki** [comment]
can we close this then?

**Comment by twicki** [comment]
now we're able to to up to 8

**Pull request closed without merging**